docs/specs/bind-unbind-workflow.md

```diff
@@ -0,0 +1,32 @@
+## Bind / Unbind New Work flow
+
+The goal of this new work flow is to give more control over the actions that the APB takes on bind and unbind to the APB author. This means that we want the APB should have the ability to manage the bindings that are requested of it.
+
+While this is still using existing workflow pieces (i.e extracting credentials, saving those credentials), it is also creating a new workflow. This is an enhancement on the current design and will allow for the APB to choose which workflow it would prefer to follow. 
+
+### Default Settings Needed
+* set `launch_apb_on_bind: true` - If this is set to false we will behave the exact same as today, handing back the credentials received at provision to the service catalog.
+
+NOTE: The launch_apb_on_bind will eventually be deprecated.
+
+
+### Brokers Responsibility
+* The broker will save the credentials for each bind. this will be saved in etcd under the `extracted_credentials` directory. The key will be the binding id.
+* The broker will be responsible for providing the provisioned credentials (extracted credentials given at provision time) to the APB as extra vars during bind and unbind. Example
```

Comment:
Line 15 is a change in the broker.
 - Pass provision credentials tto an APB duing a bind